### PR TITLE
Use a constant for content null

### PR DESCRIPTION
**Pull request by duncanspumpkin**
Fixes potential issues with ui drawing incorrect images

Anything that used StringIds::null was potentially drawing nonsene.

### CODE REVIEW

File: src/OpenLoco/Windows/StationList.cpp [hunk 1]
```diff
@@ -56,11 +56,11 @@ namespace OpenLoco::Ui::Windows::StationList
         makeRemapWidget({ 3, 15 }, { 31, 27 }, WidgetType::tab, WindowColour::secondary, ImageIds::tab, StringIds::tooltip_road_stations),
         makeRemapWidget({ 3, 15 }, { 31, 27 }, WidgetType::tab, WindowColour::secondary, ImageIds::tab, StringIds::tooltip_airports),
         makeRemapWidget({ 3, 15 }, { 31, 27 }, WidgetType::tab, WindowColour::secondary, ImageIds::tab, StringIds::tooltip_ship_ports),
-        makeWidget({ 0, 14 }, { 26, 26 }, WidgetType::buttonWithImage, WindowColour::primary, StringIds::null, StringIds::tooltip_select_company),
```
Comment (AaronVanGeffen) [inline src/OpenLoco/Windows/StationList.cpp:59]: Looks like the station list and vehicle windows were the only ones affected by the issue? Probably worth a changelog entry, mentioning them by name.

**Review by AaronVanGeffen — APPROVED** [review]

File: src/OpenLoco/Windows/StationList.cpp (hunk 1 above)
Comment (duncanspumpkin) [reply src/OpenLoco/Windows/StationList.cpp:59]: The other one was the title screen

Comment (AaronVanGeffen) [reply src/OpenLoco/Windows/StationList.cpp:59]: > The other one was the title screen

Then this would fix #293

Comment (duncanspumpkin) [reply src/OpenLoco/Windows/StationList.cpp:59]: Yes that's definitely related

**Pull request merged**